doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,168 @@
 
+e.g. Running the script for a project in `sdk\deviceupdate` would look like this: 
 ```
-sdk\<your-service-name>\Azure.<group>.<service>\src> dotnet msbuild /t:GenerateCode
+eng\scripts\Export-API.ps1 deviceupdate
 ```
 
-#### 3. Explore the Generated Code
-
-After you've run the `GenerateCode` command above, you should find a `Generated` folder in your `Azure.<group>.<service>` project.
-
-If you're interested, you can look through the swagger file to see how the autorest.csharp extension translated the REST API definition into C# code. This could help you later if you decide to make adjustments to your API design in the swagger file directly, so they can be used for all Tier 1 languages.
-
-**Learn more**: see the [OpenAPI Swagger Documentation](https://swagger.io/docs/) to learn more about swagger file formats.
+## Add package ship requirements
+You can continuously work on the first-version shippable package created above to refine, such as add Tests, Samples and refine the Readme.md and Changelog.
 
-#### 4. Save and commit your changes to your branch
+### Tests ###
 
-This will make it easier to see the changes you make when you re-generate the code later.
-
-Congratulations, you've generated your first Azure SDK library! Next, we'll discuss how we can add tests, samples and documents.
-
-## Tests
-
-In this section, we will talk about adding unit tests and live tests and how to run them. You will notice that [Azure.Template.Generated.Test](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/template-dpg/Azure.Template.Generated/tests) project contains [TemplateServiceTestEnvironment](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-dpg/Azure.Template.Generated/tests/TemplateServiceTestEnvironment.cs) file where we added live test environment to run live tests. We added live tests in [TemplateServiceLiveTests.cs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-dpg/Azure.Template.Generated/tests/TemplateServiceLiveTests.cs) file and unit tests in [TemplateServiceTests.cs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-dpg/Azure.Template.Generated/tests/TemplateServiceTests.cs) file.
+In this section, we will talk about adding unit tests and live tests and how to run them. You will notice that there is a test project under Azure.<group>.<service>\tests.
 
 Here is the step by step process to add tests:
+- Add other client parameters in `<client-name>ClientTestEnvironment.cs`
+- Update `<client-name>ClientTest.cs`. 
+  - Comment-out the 'CreateClient' method, and update the new <service>Client statement.
+  - remove all the template project tests, and write the tests according to the commented Test method template. Please refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#live-testing) to add live tests.
```

Comment:
It would be better to link to the Test Framework readme - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/README.md

Reply:
The doc as add refer to the 'live test' doc (https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#live-testing), and in the doc, it has a link to the Test Framework readme to tell customer how to write test under Test Framework.

Is it enough? or we should add the TestFramework readme link specifically again in the doc?